SEDaily-IOS/GeneralCollectionViewController.swift

```diff
@@ -184,26 +172,12 @@ class GeneralCollectionViewController: UICollectionViewController, IndicatorInfo
             return false
         }
         
-        // Format last item date
-        let formatterTmp = DateFormatter()
-        formatterTmp.dateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXXXX"
-        let lastItemDateObject = formatterTmp.date(from: lastItemDate!)
-        if lastItemDateObject == nil {
-            return false
-        }
-        formatterTmp.dateFormat = "yyyy-MM-dd"
```

Comment:
We can't remove this formatting. The key needs to be hashed with out the hour, minutes, seconds, etc.